index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Microverse Partnership: Ademola-Agasha</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="container">
```

Comment:
Please remove this div element. I suggest that you follow the semantic HTML5 page structure. This will makes your web pages more informative and adaptable, allowing browsers and search engines to better interpret content.